docs/view_components/welcome.rst

```diff
@@ -1,11 +1,47 @@
-Component title
----------------
+.. _welcome-label:
+
+Welcome page
+------------
+
+The welcome window serves as an orientation point for the user and let them know, in as brief words (and other audiovisual content) as possible, what to expect from the tool. It is also the launch point from where the user begins their energy systems simulation.
+
 
 Attributes
 ^^^^^^^^^^
 
+#. **Welcome message**
+    * A text
+
+#. **Video intro message**
+    * A short, 3 minute video clip that provides the important information such as tool features, capabilities, where to find help, etc., very briefly
+    * This clip is embedded into the welcome pop-up. The user can view it on YouTube (or Vimeo) on a separate tab of the browser by clicking on a link on the video
+
+#. **Check-box to disable the welcome pop-up**
+
+#. **Hyperlinks**
+    * Clickable links to read more, documentation, examples, use-cases, video tutorials, code-base (GitHub repo) and contact us
+
 Actions
 ^^^^^^^
 
+* Pop-up that springs up everytime the software is started
+* Checking the tick-box would disable the pop-up from appearing everytime the software is launched
+* The user could later re-enable the appearance of the pop-up window through *Display* or *Preferences* menu in *Settings*
+* Clicking on the hyperlinks will open the respective topic in a separate tab on the browser (e.g.: GitHub repo)
+
+Requirement
+^^^^^^^^^^^
+
+* If the user ticks the check-box *Do not show again*, the pop-up should not appear during the subsequent launch of the tool
+
+Link with other view-components
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Only connected to the landing view, which appears everytime the tool is launched
```

Comment:
I would say it is linked to the file menu view component (from there you can make the welcome view-component appear again)

Reply:
will add your suggestion to the links section